modules/gapi/include/opencv2/gapi/gcomputation.hpp

```diff
@@ -528,6 +531,7 @@ class GAPI_EXPORTS_W GComputation
         GCompileArgs comp_args = std::get<sizeof...(Ts)-1>(meta_and_compile_args);
         return compileStreaming(std::move(meta_args), std::move(comp_args));
     }
+    void recompile(GMetaArgs&& in_metas, GCompileArgs &&args);
```

Comment:
Make it `private` please. Users shouldn't call it as there's no proper flow defined and documented for this function.

Reply:
It's `protected`